e2e-tests/portal/cypress/e2e/journeys/maker/dashboard/fixtures/deals.js

```diff
@@ -38,8 +38,12 @@ const GEF_DEAL_MAKER_2 = {
 
 const GEF_DEAL_BANK_2_MAKER_2 = {
   ...GEF_DEAL,
+  exporter: {
+    ...GEF_DEAL.exporter,
+    companyName: 'Mock company name',
```

Comment:
is this necessary? it looks to be the same as above

Reply:
Not sure. @ttbarnes is this needed?

Reply:
Good spot - no it is not needed. We don't need to define `exporter` here at all since it is using `GEF_DEAL`.

I suggest reverting the change to `bankInternalRefName` below though.

Reply:
reverted back